inspire/dojson/hep/schemas/hep-0.0.1.json

```diff
@@ -530,26 +528,32 @@
             },
             "type": "array"
         },
-        "experiment": {
-            "required": [
-                "value"
-            ],
-            "type": "object",
-            "properties": {
-                "curated_relation": {
-                    "default": false,
-                    "type": "boolean",
-                    "description": "Was the experiment actually proofchecked by a cataloguer?"
-                },
-                "recid": {
-                    "type": "integer",
-                    "description": "Record ID of the corresponding experiment."
-                },
-                "value": {
-                    "type": "string",
-                    "description": "Raw string. Do we actually need it at all? Can it be taken from the record ID? NOTE: I am not adding accelerator field. Do we have papers on accelerators but not on experiments? Can we always derive the accelerator from the experiment?"
+        "accelerator_experiments": {
+            "uniqueItems": true,
+            "items": {
+                "type": "object",
+                "properties": {
+                    "curated_relation": {
+                        "default": false,
+                        "type": "boolean",
+                        "description": "Was the experiment actually proofchecked by a cataloguer?"
+                    },
+                    "recid": {
+                        "type": "integer",
+                        "description": "Record ID of the corresponding experiment."
+                    },
+                    "experiment": {
```

Comment:
There are papers on accelerator physics not referring to a specific experiment.
For searching it's maybe useful to have the accelerator in the record itself even for experimental papers.
And sometimes experimental papers - esp very old ones - don't have an experiment name. We just know that the experiment has been done at a specific accelerator.

- Annette

On Oct 7, 2015, at 10:11 AM, Javier Martin Montull <notifications@github.com<mailto:notifications@github.com>> wrote:


In inspire/dojson/hep/schemas/hep-0.0.1.json<https://github.com/inspirehep/inspire-next/pull/400#discussion_r41362236>:

> -                    "description": "Raw string. Do we actually need it at all? Can it be taken from the record ID? NOTE: I am not adding accelerator field. Do we have papers on accelerators but not on experiments? Can we always derive the accelerator from the experiment?"
> +        "accelerator_experiments": {
> +            "uniqueItems": true,
> +            "items": {
> +                "type": "object",
> +                "properties": {
> +                    "curated_relation": {
> +                        "default": false,
> +                        "type": "boolean",
> +                        "description": "Was the experiment actually proofchecked by a cataloguer?"
> +                    },
> +                    "recid": {
> +                        "type": "integer",
> +                        "description": "Record ID of the corresponding experiment."
> +                    },
> +                    "experiment": {


@kaplun<https://github.com/kaplun> As this is a relationship, should experiment, accelerator and institution be part of the HEP record or this should be in the corresponding Experiment record?

—
Reply to this email directly or view it on GitHub<https://github.com/inspirehep/inspire-next/pull/400/files#r41362236>.